src/components/PopoverMenu.tsx

```diff
@@ -247,7 +256,7 @@ function PopoverMenu({
             restoreFocusType={restoreFocusType}
         >
             <FocusTrapForModal active={isVisible}>
-                <View style={isSmallScreenWidth ? {} : styles.createMenuContainer}>
+                <ScrollView style={isSmallScreenWidth ? {} : styles.createMenuContainer}>
```

Comment:
We are looking to do something similar [here](https://github.com/Expensify/App/pull/49051) but we want the scrollview to exclude the title (title should be fixed). We will move this ScrollView to wrap only `currentMenuItems`. Let us know if you see a reason not to do so.